src/displayUI.js

```diff
@@ -114,20 +118,7 @@ export function searchLinks() {
     e.preventDefault();
     document.querySelector("#oneCategoryDisplayBox").innerHTML = "";
 
-    instanceOfMainClass.searchAllLinks(e).forEach(el => {
-      displayOneLink(el);
-    })
-  });
-}
-
-export function searchChosenLinksOnly() {
-  const searchBarLinks = document.querySelector("#searchBarLinks");
-
-  searchBarLinks.addEventListener("input", (e) => {
-    e.preventDefault();
-    document.querySelector("#oneCategoryDisplayBox").innerHTML = "";
-
-    instanceOfMainClass.searchChosenLinks(e).forEach(el => {
+    instanceOfMainClass.getAllLinks(instanceOfMainClass.filtered, e).forEach(el => {
```

Comment:
getAllLinks not need param filtered, since is in same object class, and could acess to to prop, basic OOP principles, getAllLinks could access all class properties/methods.